solutions/logging/client-experimentation/README.md

```diff
@@ -0,0 +1,136 @@
+<!-- BEGINNING OF PRE-COMMIT-BLUEPRINT DOCS HOOK:TITLE -->
+# client-experimentation-logging-package
+
+
+<!-- END OF PRE-COMMIT-BLUEPRINT DOCS HOOK:TITLE -->
+**`Note:`** The [core-experimentation](../core-experimentation/) package must be deployed before deploying the `client-experimentation` logging package.
+
+<!-- BEGINNING OF PRE-COMMIT-BLUEPRINT DOCS HOOK:BODY -->
+Package to deploy client-experimentation logging solution
+
+## Setters
+
+|           Name           |           Value           | Type | Count |
+|--------------------------|---------------------------|------|-------|
+| client-displayname       | Client1                   | str  |     2 |
+| client-folderviewer      | group:client1@example.com | str  |     0 |
```

Comment:
doesn't look like the `client-folderviewer` is used, could possibly be removed from setters.yaml?